firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/NewReleaseFetcher.java

```diff
@@ -195,6 +203,21 @@ private long getInstalledAppVersionCode(Context context) throws FirebaseAppDistr
     return PackageInfoCompat.getLongVersionCode(pInfo);
   }
 
+  private String getInstalledAppVersionName(Context context)
+      throws FirebaseAppDistributionException {
+    PackageInfo pInfo;
+    try {
+      pInfo = context.getPackageManager().getPackageInfo(context.getPackageName(), 0);
+    } catch (PackageManager.NameNotFoundException e) {
+      LogWrapper.getInstance().e(TAG + "Unable to locate Firebase App.", e);
```

Comment:
We're not looking for the "Firebase App" right? Seems like "Package name not found when looking for installed app version name" or something would be more accurate.

Reply:
We should probably update the error message in the `getInstalledAppVersionCode` method as well

Reply:
That sounds right. Tweaked the logs to better reflect what is going on.